core/src/epicli/data/common/ansible/playbooks/roles/kubernetes_master/tasks/cni-plugins/canal.yml

```diff
@@ -1,4 +1,39 @@
 ---
+- name: Configure NetworkManager before attempting to use Canal networking
+  block:
+    # /etc/NetworkManager/conf.d directory is created by NetworkManager(rpm)/network-manager(deb) package
+    - name: Check if /etc/NetworkManager/conf.d exists
+      stat:
+        path: /etc/NetworkManager/conf.d
+        get_attributes: no
+      register: network_manager_conf_d_stat
+
+    # Remove flannel.1 interface from NetworkManager managed devices
+    - name: Copy flannel.conf to /etc/NetworkManager/conf.d
+      template:
+        src: network-manager.conf.j2
```

Comment:
can the file be named `flannel.conf.j2` if it's `flannel.conf`?

Reply:
@mkyc network-manager.conf.j2 is a base template for NetworkManager configuration all CNI plugins. Keeping a separated template for each CNI is redundant.